client/src/api/plugin/Queries.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { PluginDefinition } from '../Models';
+
+export function useGetPlugins() {
+  return useQuery<PluginDefinition[]>(
+    ['plugins'],
+    async () => {
+      const response = await axios.get<PluginDefinition[]>('/api/plugins').catch((err) => {
+        console.log('useGetPlugins error', err);
```

Comment:
still need this?

Reply:
This si just we have some feedback in case an error happens the happy path has no logs we whoudl probably look ot a better log and alert system